src/scripts.js

```diff
@@ -357,3 +357,9 @@ function createInstructionList(recipe) {
 function sortRecipesByName() {
   recipeRepository.recipes.sort((a, b) => a.name - b.name);
 }
+
+function clearTags() {
+  const selectedTags = document.querySelectorAll('.tag-selected')
```

Comment:
This looks great - thank you for setting this up! I noticed this is manipulating the DOM too, could some or all of this function also be moved over?

Reply:
Great point! I will move the DOM manipulation pieces.